Week1/homework/aboutMe/index.js

```diff
@@ -0,0 +1,25 @@
+let body = document.querySelector("BODY");
+//document.getElementById("body").style.fontFamily = "Impact,Charcoal,sans-serif";
+
+body.style.fontFamily = "Arial, sans-serif";
+
+
+document.getElementById("nickname").textContent = "Kotsidas";
+document.getElementById("fav-food").textContent = "Pasta";
+document.getElementById("hometown").textContent = "Heraclion";
+
+let list = document.querySelectorAll("li")
+ 
+for (let i=0; i<list.length; i++){
+  list[i].classList.add("list-item")
+}
```

Comment:
It is perfectly fine as it is. Personal preference for loops is .forEach. I try to avoid for loops because you cannot understand straight away what is going on